features/collection/thresholding/impl/src/main/java/org/opennms/netmgt/threshd/AbstractThresholdEvaluatorState.java

```diff
@@ -151,15 +156,30 @@ public String toString() {
 
         this.thresholdingSession = thresholdingSession;
         kvStore = getKvStoreForType(stateType, thresholdingSession.getBlobStore());
-        key = String.format("%d-%s-%s-%s-%s-%s", thresholdingSession.getKey().getNodeId(),
+        key = String.format("%d-%s-%s-%s-%s-%s-%d", thresholdingSession.getKey().getNodeId(),
```

Comment:
Kept other parts of key same as it may help debugging to have them descriptive.

Reply:
Makes sense 👍🏻 . I guess the downside is we won't have a fixed width key like we would with a hash. But maybe having it human readable is more valuable.